### ISSUE TITLE
Remove hardcoded artran cashdrawer value (currently 'ZZ')

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1749]
_Imported from trac ticket 1749.
Created by **kbower**
Opened in trac: Jul 4, 2012, 3:00:17 PM
Last modified in trac: Jul 24, 2012, 11:30:06 AM_
***
_by kbower:_
After successful funding when creating AR_TRN populate CSH_DRW_CD with value of 'ZZ'.

Other values like mediumtypeid = u'NOR' and paymenttypeid=u'NWF' also might be el dorado specific?

This should be system parameter or rule controlled.

**Comment by kbower** [comment]
_**sshah** trac comment on Jul 6, 2012, 3:47:17 PM:_

(Changeset [[2842]](../commit/790ed5025869766bee2325121b8c978c37fc9dba))
 * Fixes #1749 default cash drawer to ZZ when funding through wells fargo for eldorado.

**Comment by kbower** [comment]
_**sshah** trac status change on Jul 6, 2012, 3:47:17 PM:_ **closed**

**Comment by kbower** [comment]
_**kbower** trac comment on Jul 7, 2012, 9:55:23 PM:_

(Changeset [[2843]](../commit/f0063ec80d06800345d5a1c315dc3fea3ba6e89d))
 * revert [r2842](../commit/790ed5025869766bee2325121b8c978c37fc9dba) re #1749 With growing number of clients, we can't be branching source code based on the customer; that's what the rules framework and system parameters are all about.

**Comment by kbower** [comment]
_**kbower** trac comment on Jul 7, 2012, 10:01:06 PM:_

This isn't a huge priority; we just don't want it there long term.  Solution should be system parameter (or rules) based.  If rules based, invoke a function that the framework does "pass".  Then in a rule, eld can do this:
```python
transaction.cashdrawer = u'ZZ' 
transaction.paymenttypeid=u'NWF'

financetransaction.mediumtypeid = u'NOR'
```
Aren't all those el dorado specific data updates?

**Comment by kbower** [comment]
_**kbower** trac status change on Jul 7, 2012, 10:01:06 PM:_ **reopened**

**Comment by kbower** [comment]
_**sshah** trac comment on Jul 24, 2012, 11:30:06 AM:_

(Changeset [[2913]](../commit/1b8338395289400f36b39a3d979a45c590c68ffb))
 * Fixes #1749 Wells Fargo funding transactions default value (CashDrawer=ZZ)

**Comment by kbower** [comment]
_**sshah** trac status change on Jul 24, 2012, 11:30:06 AM:_ **closed**
